src/components/map-overlays/map-overlays.component.tsx

```diff
@@ -164,6 +168,23 @@ const MapOverlays = ({
             ))}
         </>
       ) : null}
+
+      {/**
+       * Adds a marker for startLocation and endLocation
+       */}
+      {travelState === TravelState.TRAVELLING
+        ? [startLocation, endLocation].map((marker, index) => (
+            <CustomMarker
+              testID={index === 0 ? "startLocation" : "endLocation"}
+              markerType={index === 0 ? "startLocation" : "endLocation"}
+              location={marker.location}
+              // Temporary. Setting id to be the displayname would cause bugs.
```

Comment:
What exactly is temporary? Isn't `text={marker.displayName}` what is intended?

Reply:
Sorry, this was Cristian and I having a conversation through code. Will this clean this up right now🚀

Reply:
o7